shortstories/templates/shortstories/shortstories_edit.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base/index.html' %}
+{% block title %}
+    NewShortStories
```

Comment:
Create a new short story. (this going to be used in your title)

Reply:
done